scripts/run_e2e_tests.sh

```diff
@@ -46,6 +46,7 @@ if [ "$platform" = "ios" ]; then
 elif [ "$platform" = "android" ]; then
   app_id=$(parse_env_variable ANDROID_APP_ID ".env.$env")
   echo "Running Android tests on $env environment with app id: $app_id"
+  adb reverse "tcp:8081" "tcp:8081" # bind metro server
```

Comment:
je propose de se passer de commentaire en explicitant

```suggestion
  METRO_SERVER_PORT="8081"
  adb reverse "tcp:${METRO_SERVER_PORT}" "tcp:${METRO_SERVER_PORT}"
```

(oui, je sais que j'aurais pu l'écrire correctement dès le départ =P )